pgsync/sync.py

```diff
@@ -433,6 +433,12 @@ def should_skip_event(self, event: dict) -> bool:
 
         logger.debug(f"should_skip_event: validating event: {event}")
 
+        if event['indices'] is not None:
+            # event['indices'] set to None when pgsync parse replication slot
+            if self.index in [event['indices']]:
```

Comment:
skipping if self.index is **_IN_** the event['indices']?

Reply:
No, event['indices'] should be enumerate, can replace with enumerate(event['indices'])